README.md

```diff
@@ -1,51 +1,99 @@
-# Hammer.js 2.0.6
+# [hammer.js][hammerjs-url]  [![NPM Version][npm-image]][npm-url]  [![NPM Downloads][downloads-image]][downloads-url]  [![Build Status][travis-image]][travis-url]
+> A JavaScript library for detecting touch gestures.
 
-[![Build Status](https://travis-ci.org/hammerjs/hammer.js.svg)](https://travis-ci.org/hammerjs/hammer.js)
 
-## Support, Questions, and Collaboration
+## Installation
+### NPM
+```sh
+npm install --save hammerjs
+```
 
-[![Slack Status](https://hammerjs.herokuapp.com/badge.svg)](https://hammerjs.herokuapp.com/)
+**or**
 
-## Documentation
+### Yarn
+```sh
+yarn add hammerjs
+```
+
+**or**
+
+### CDN
+[https://cdnjs.com/libraries/hammer.js/](https://cdnjs.com/libraries/hammer.js/)
 
-Visit [hammerjs.github.io](http://hammerjs.github.io) for detailed documentation.
 
+## Usage
 ```js
-// get a reference to an element
-var stage = document.getElementById('stage');
+// Get a reference to an element
+var square = document.querySelector('.square');
 
-// create a manager for that element
-var mc = new Hammer.Manager(stage);
+// Create a manager to manager the element
+var manager = new Hammer.Manager(square);
 
-// create a recognizer
+// Create a recognizer
 var Rotate = new Hammer.Rotate();
 
-// add the recognizer
-mc.add(Rotate);
+// Add the recognizer to the manager
+manager.add(Rotate);
 
-// subscribe to events
-mc.on('rotate', function(e) {
-    // do something cool
-    var rotation = Math.round(e.rotation);    
-    stage.style.transform = 'rotate('+rotation+'deg)';
+// Subscribe to desired event
+manager.on('rotate', function(e) {
+    var rotation = Math.round(e.rotation);  
+   
+    square.style.transform = 'rotate(' + rotation + 'deg)';
+    console.log("You're rotating me!");  
 });
 ```
```

Comment:
Maybe also inform about the 'quick usage', just `new Hammer()` and you're good to go.